x-pack/plugins/security_solution/public/resolver/view/panels/process_event_list_narrowed_by_type.tsx

```diff
@@ -97,6 +97,7 @@ export const ProcessEventListNarrowedByType = memo(function ProcessEventListNarr
 }) {
   const processName = processEvent && event.eventName(processEvent);
   const processEntityId = event.entityId(processEvent);
+

```

Comment:
I think more than just this file have a file name that doesn't line up with the component name. Would probably make sense to fix those as well

Reply:
I'm changing them as I find 'em. I don't think we need a rule that says '1:1 relationship between components and filenames.' That said, if a file has just one export, which is a component, it would be nice to name the component and the file the same.